client/src/catalog-app.html

```diff
@@ -592,8 +592,14 @@ <h2 id="domain-title">webcomponents.org</h2>
       _notifyUser: function(event) {
         var options = event.detail;
         this.$.toast.text = options.message;
-        this.$.toastAction.textContent = options.buttonText;
+        if (options.hasOwnProperty('buttonText')) {
+          this.$.toastAction.removeAttribute('hidden');
```

Comment:
The reason I added/removed hidden was because even if the buttonText was empty, the extra space is still going to be there and it looks weird:

![image](https://cloud.githubusercontent.com/assets/2008881/25233392/47402302-25a4-11e7-8862-6b2234333d3c.png)

Would you like me to try an alternative?